### ISSUE TITLE
[Header/body newsletter list conflicts] Thank you page

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#3928]
### Issue:
Add the paragraph giving people the option to manage their newsletter subscriptions in case they signed up for past Mozilla newsletters and want to update their preferences (just because we have so many of them!). It is also a way for them to remember past subscriptions.

### Screenshot:
<img width="226" alt="image" src="https://user-images.githubusercontent.com/22157921/69086949-668a4380-09fb-11ea-9270-813caaa25159.png">

### Mockup:
https://mozilla.invisionapp.com/share/8TUW2XEGAF3#/393965606_Final-Proposal-Thanks-Page

### Link:
https://www.mozilla.org/en-US/newsletter/recovery/

### IMPORTANT: The copy I currently have in the mockup is just a placeholder.

**Comment by kristinashu** [comment]
Please work with Jen to finalize copy and give Steph McV a heads up that we will be updating it.

**Comment by taisdesouzalessa** [comment]
Jen is on PTO until Wed. Discussion is blocked by that atm. Plan is to ping her on Thursday.

**Comment by taisdesouzalessa** [comment]
Jen's new copy:

"Thanks!

One more thing. To protect you from unwanted emails, we ask you to confirm your opt-in to our email subscriptions. If you haven’t previously confirmed your opt-in to a Mozilla-related email subscription you may have to do so now. Please check your inbox or spam filter for an email from us to click and confirm your subscription. 

If you have already confirmed your opt-in to receive Mozilla-related emails, you can now manage your subscriptions and update your email preferences."

[Original doc here](https://docs.google.com/document/d/1BS493HcyIt34cTRYWGFVbzqZidpaUpCCToDI1XI2nNs/edit)

I really like the fact she added the reasoning on why we need the double opt-in, from the audience's perspective. This copy seems good to me. 

@kristinashu let me know what you think and if anybody else should be involved on the approval of it. Also, this "thank you" message is in all components that have a form to subscribe to email. Should we change them all? It may require a new ticket if we do.

**Comment by kristinashu** [comment]
Thanks @taisdesouzalessa! You could run it past Steph McV to see if she has any feedback. Once copy is final then please update the comp and open the implementations tickets.

**Comment by taisdesouzalessa** [comment]
Implementation ticket here: #4143. Closing the design ticket.
